### PR TITLE
Feat/forms openapi

### PR DESCRIPTION
**Pull request by kajm**
## Explain the changes you’ve made
Created openapi docs for forms api

## Explain why these changes are made
We want our api to be documented, hence creating the openapi.yml file.

## How to test

1. Checkout this branch
2. Open the forms-api in Vscode
3. Open openapi.yml file from swagger

### CODE REVIEW

**Comment by helsingborgsstad** [comment]
Task linked: [CU-3xcre1x Skapa OpenAPI för forms service](https://app.clickup.com/t/3xcre1x)

File: services/forms-api/openapi.yml [hunk 1]
```diff
@@ -0,0 +1,464 @@
...
+      tags:
+        - Forms
+      parameters:
+        - name: formId
+          in: path
+          required: true
+          schema:
+            type: string
+      summary: Updates a form.
+      description: This endpoint updates a current form.
```
Comment (MasterKenth) [inline services/forms-api/openapi.yml:92]: Might add that if `formId` does not point to an existing form a new one is actually created (possibly a bug in forms-api but that's what it does).

File: services/forms-api/openapi.yml [hunk 2]
```diff
@@ -0,0 +1,464 @@
...
+      tags:
+        - Forms
+      parameters:
+        - name: formId
+          in: path
+          required: true
+          schema:
+            type: string
+      summary: Delete form.
+      description: This endpoint deletes a form from database.
```
Comment (MasterKenth) [inline services/forms-api/openapi.yml:120]: Might add that this silently succeeds (status `200`) even if `formId` is invalid (again - might be a bug/unintended behavior in forms-api but that's what it does).

**Review by MasterKenth — CHANGES_REQUESTED** [review]

File: services/forms-api/openapi.yml [hunk 3]
```diff
@@ -0,0 +1,464 @@
...
+            schema:
+              $ref: '#/components/schemas/Form'
+      responses:
+        '201':
+          description: OK.
+          content:
+            'application/json':
+              schema:
+                $ref: '#/components/schemas/GetFormResponse'
+        '403':
```
[the remark this answers is not in the document]
Comment (kajm) [reply services/forms-api/openapi.yml:55]: fixed

File: services/forms-api/openapi.yml [hunk 4]
```diff
@@ -0,0 +1,464 @@
...
+        - name: formId
+          in: path
+          required: true
+          schema:
+            type: string
+      summary: Get form.
+      description: This endpoint returns a form-builder form by its unique id.
+      security:
+        - ApiKeyAuth: []
+      responses:
```
[the remark this answers is not in the document]
Comment (kajm) [reply services/forms-api/openapi.yml:72]: fixed

File: services/forms-api/openapi.yml (hunk 1 above)
Comment (kajm) [reply services/forms-api/openapi.yml:92]: fixed

File: services/forms-api/openapi.yml (hunk 2 above)
Comment (kajm) [reply services/forms-api/openapi.yml:120]: fixed

File: services/forms-api/openapi.yml [hunk 5]
```diff
@@ -0,0 +1,530 @@
...
+      summary: Delete form.
+      description: This endpoint deletes a form from database.
+      security:
+        - ApiKeyAuth: []
+      responses:
+        '200':
+          description: |
+            OK.
+
+            <b>Note</b>
```
Comment (MasterKenth) [inline services/forms-api/openapi.yml:141]: Markdown is supported and should be more preferable here unless there is a reason to use html?

Comment (kajm) [reply services/forms-api/openapi.yml]: fixed

**Review by MasterKenth — APPROVED** [review]

**Pull request merged**